service/framework.go

```diff
@@ -173,6 +150,95 @@ func newCRDFramework(config Config) (*framework.Framework, error) {
 		}
 	}
 
+	versionedResources, err := NewVersionedResources(config, k8sClient, awsConfig, awsHostConfig)
```

Comment:
Creating the map of resources by version is now extracted to `NewVersionedResources`.